libopenage/engine/engine.h

```diff
@@ -14,12 +16,18 @@ namespace std {
 class jthread : public thread {
 public:
 	using thread::thread; // needed constructors
```

Comment:
maybe we can define a preprocessor macro that detects when clang supports jthreads -> we notice the error and can get rid of this instead of forgetting. then again it doesn't hurt having.

Reply:
good idea. I've added a pre-processor `#error`.